{"levelname": "INFO", "asctime": "2022-11-28T22:50:50.028Z", "name": "root", "message": "Unable to find a row with primary key 5771a87e-1283-4518-88a0-0a35d0a6738f, in Collection table.", "lineno": 43, "pathname": "/single-cell-data-portal/backend/common/entities/entity.py"}
